components/content/simple-editor/CustomTresOrbitControls.vue

```diff
@@ -0,0 +1,35 @@
+<script setup lang="ts">
+import { extend, useTresContext } from '@tresjs/core'
```

Comment:
Interesting why your custom OrbitControls and not the Cientos one?

Reply:
without it I get an error that I still don't fully understand how to solve. I'm not sure if it is my fault or a bug from tresjs, I spent a good amount of time trying to figure out but had to move on.